include/podio/detail/Link.h

```diff
@@ -147,11 +145,9 @@ class LinkT {
   /// @note All setFrom overloads are equivalent and the correct one is selected
   /// at compile time. We need to differentiate between the handles, only to
   /// make the python bindings work
-  template <typename FromU,
-            std::enable_if_t<Mutable && std::is_same_v<detail::GetMutableHandleType<FromT>, FromU> &&
-                                 detail::isMutableHandleType<FromU>,
-                             int> = 0>
-  void setFrom(FromU value) {
+  template <typename FromU>
+  requires(Mutable&& std::is_same_v<detail::GetMutableHandleType<FromT>, FromU>&&
+               detail::isMutableHandleType<FromU>) void setFrom(FromU value) {
```

Comment:
```suggestion
  template <typename FromU>
  requires(Mutable && std::is_same_v<detail::GetMutableHandleType<FromT>, FromU>&&
               detail::isMutableHandleType<FromU>) 
  void setFrom(FromU value) {
```

Does this formatting get past clang-format? I think this would be much more readable than the current form.

Reply:
Changed the format to C++20 and it looks better now